www/content/examples/lazy-load.md

```diff
@@ -46,14 +46,14 @@ img {
     init("/demo", function(request, params){
       return lazyTemplate();
     });
-    
+
     onGet("/graph", function(request, params){
       return "<img alt='Tokyo Climate' src='/img/tokyo.png'>";
     });
-    
+
     // templates
     function lazyTemplate(page) {
-      return `<div hx-get="/graph" hx-trigger="load">
+      return `<div hx-get="/graph" aria-busy='true' hx-trigger="load">
```

Comment:
`aria-busy` is useless without `aria-live`.

Reply:
removed